azurerm/internal/services/apimanagement/registration.go

```diff
@@ -43,6 +43,7 @@ func (r Registration) SupportedResources() map[string]*pluginsdk.Resource {
 		"azurerm_api_management_api_version_set":             resourceApiManagementApiVersionSet(),
 		"azurerm_api_management_authorization_server":        resourceApiManagementAuthorizationServer(),
 		"azurerm_api_management_backend":                     resourceApiManagementBackend(),
+		"azurerm_api_management_cache":                       resourceApiManagementCache(),
```

Comment:
is this always redis?
```suggestion
		"azurerm_api_management_redis_cache":                       resourceApiManagementRedisCache(),
```